### ISSUE TITLE
Import IFC with the last version of blenderBIM (blenderbim-230506-py310-macosm1)

### ISSUE THREAD

**Issue by AGGarcia45** [IfcOpenShell/IfcOpenShell#3351]
I'm trying to open an IFC file using the last version of blenderBIM - version 3.5.1 (2023-04-25) and the last add-on BlenderBIM 0.0.230506 for macosm1 and I've received the follow error message: 

```
Python: Traceback (most recent call last):
  File "/Users/Anderson/Library/Application Support/Blender/3.5/scripts/addons/blenderbim/bim/module/project/operator.py", line 594, in execute
    ifc_importer.execute()
  File "/Users/Anderson/Library/Application Support/Blender/3.5/scripts/addons/blenderbim/bim/import_ifc.py", line 241, in execute
    self.create_styles()
  File "/Users/Anderson/Library/Application Support/Blender/3.5/scripts/addons/blenderbim/bim/import_ifc.py", line 1463, in create_styles
    self.create_style(style, blender_material)
  File "/Users/Anderson/Library/Application Support/Blender/3.5/scripts/addons/blenderbim/bim/import_ifc.py", line 1491, in create_style
    self.create_surface_style_rendering(blender_material, surface_style)
  File "/Users/Anderson/Library/Application Support/Blender/3.5/scripts/addons/blenderbim/bim/import_ifc.py", line 1516, in create_surface_style_rendering
    bsdf = blender_material.node_tree.nodes["Principled BSDF"]
KeyError: 'bpy_prop_collection[key]: key "Principled BSDF" not found'
```

Can someone help-me with this issue? 

Thanks in advance.

Anderson

**Comment by Moult** [comment]
Is your Blender set to English?

**Comment by AGGarcia45** [comment]
> Is your Blender set to English?

No! Is set to Portuguese (Brazil)

**Comment by AGGarcia45** [comment]
I've reset to English and works! 
Tks!

**Comment by Moult** [comment]
Let's leave this open until we fix it :)

**Comment by Gorgious56** [comment]
I think this issue was solved in one of the recent commits, wasn't it ? https://github.com/IfcOpenShell/IfcOpenShell/blob/8f18db64cf48c4685de4e3f0cdf7047de2dce67a/src/blenderbim/blenderbim/tool/blender.py#L190-L198

**Comment by Andrej730** [comment]
@Gorgious56 you're right, it was fixed in a34be6a
